### PR TITLE
Ux 151235375 refactor

### PR DESCRIPTION
**Pull request by pierreOSI**
### Pivotal Story

- https://www.pivotaltracker.com/story/show/151235375

### Background

First part of a refactor of all labels and minor layout tweaks.

### Questions for Reviewer

Not sure why it can't be merged automatically, there might be conflict with some of the fields touching other WIP.

### Notes for Reviewer

I did refactor some fields outside of the cards mentioned (Like communication method)  because it was easier / useful (since those will be refactored too)

### Testing Notes

Passed all tests

### CODE REVIEW

**Comment by pierreOSI** [comment]
TEST PASSED

**Comment by ThomasOSI** [comment]
BTW: Code Climate grouchiness is all [similar code stuff](https://codeclimate.com/github/ca-cwds/intake/compare/ux_151235375_refactor)

**Comment by pierreOSI** [comment]
Thanks for letting me know, I do not have access to it but I was told it wasn't very adapted to React.

________________________________

File: spec/javascripts/components/screenings/allegations/ScreeningPageAllegationSpec.jsx  [hunk 1]
```diff
@@ -1,227 +0,0 @@
-import * as IntakeConfig from 'common/config'
-import Immutable from 'immutable'
-import React from 'react'
-import {ScreeningPage} from 'screenings/ScreeningPage'
-import {mount, shallow} from 'enzyme'
-import {requiredProps, requiredScreeningAttributes} from '../ScreeningPageSpec'
-
-describe('ScreeningPage', () => {
```
Comment (jgedeon-osi) [inline spec/javascripts/components/screenings/allegations/ScreeningPageAllegationSpec.jsx ]: It looks like this file was removed. Is that correct? If so why was it removed?

File: spec/javascripts/components/screenings/relationships/RelationshipCardSpec.jsx  [hunk 2]
```diff
@@ -1,266 +0,0 @@
-import * as Immutable from 'immutable'
-import RelationshipsCard from 'screenings/RelationshipsCard'
```
Comment (jgedeon-osi) [inline spec/javascripts/components/screenings/relationships/RelationshipCardSpec.jsx ]: Not sure why git says this file was deleted, but wanted to make sure it wasn't.

Comment (pierreOSI) [reply spec/javascripts/components/screenings/relationships/RelationshipCardSpec.jsx ]: Those were duplicated files apparently!

File: spec/javascripts/components/screenings/allegations/ScreeningPageAllegationSpec.jsx  (hunk 1 above)
Comment (AmanShurafa) [reply spec/javascripts/components/screenings/allegations/ScreeningPageAllegationSpec.jsx ]: Yes, this is a duplicated file with a space after the extension

File: spec/javascripts/components/screenings/relationships/RelationshipCardSpec.jsx  (hunk 2 above)
Comment (AmanShurafa) [reply spec/javascripts/components/screenings/relationships/RelationshipCardSpec.jsx ]: Yes, this is a duplicated file with a space after the extension

**Pull request closed without merging**